src/ctia/http/server.clj

```diff
@@ -185,6 +186,9 @@
             (when-let [lifetime (:lifetime-in-sec jwt)]
               {:jwt-max-lifetime-in-sec lifetime}))))
 
+         true
```

Comment:
You should probably check the configuration value here.
Otherwise this will be confusing to see the middleware is not used depending on the configuration elsewhere.

Reply:
Done.